src/PrestaShopBundle/Resources/config/services/core_services.yml

```diff
@@ -3,3 +3,4 @@ imports:
     - { resource: core/configuration.yml }
```

Comment:
I don't know if you were of it, but you could do

- { resource: core/*.yml }

instead.

Don't change it here, we'll do another pull request to improve all imports later.